docs/DeveloperGuide.md

```diff
@@ -242,143 +273,152 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 (For all use cases below, the System is the `CLITutorsBook` and the Actor is the User, unless otherwise specified)
 
 #### Use case (UC01): Add a Tutor
+{:.no_toc}
 
 **MSS**
 
 1. User keys in the tutor's details
 2. Tutor is added to the database
-   Use case ends.
+   <br>Use case ends.
 
 **Extensions**
 
 * 1a. User keys in an incorrect format.
     * 1a1. System displays an error message to tell the user about the format error.
-      Use case ends.
+      <br>Use case ends.
 
 #### Use case (UC02): Add a Student
+{:.no_toc}
 
 **MSS**
 
 1. User keys in the student's details
 2. Tutor is added to the database
-   Use case ends.
+   <br>Use case ends.
```

Comment:
Is it necessary for us to have a line break before Use case ends? I think in the textbook they also didn't have a extra line break

Reply:
Hihi, without the line break, the "Use case ends" trails behind the previous line on the website, like this: 
![image](https://user-images.githubusercontent.com/46766154/138246305-1afbb8be-3eba-40d1-8b6d-5f8ea03a5fd7.png)
So I find it necessary to add it :').